tests/management/test_finalize_xblockskill_tags.py

```diff
@@ -77,7 +77,6 @@ def test_finalize_xblockskill_tags_below_minimum_votes(self):
                 ]
```

Comment:
Same here. The `call_command` here is still using the earlier options.

Reply:
Updated. Thanks!